controller/scripts/lint.ini

```diff
@@ -25,9 +13,12 @@ disable=
  # easy:
  raise-missing-from, # W - happened only on Jenkins, add "from" after some of the "raise"
  redefined-builtin, redefined-outer-name, # W, W - colliding names: e.g. type, e.g. config
- no-self-use, # R - some functions should be made static
+ unspecified-encoding, # W - add explicit encoding="utf-8" argument when calling "open"
+ consider-using-from-import, # R - redundant "as" after "import"
+ consider-using-with, # R - use "with" instead of "acquire" + "release"
```

Comment:
do we have tickets to remove the `disable` ones?

Reply:
I opened CSI-4374